bundle/Resources/config/services/layout_resolver.yaml

```diff
@@ -29,16 +29,31 @@ services:
         tags:
             - { name: netgen_layouts.target_type.doctrine_handler, target_type: sylius_taxon_product }
 
+    netgen_layouts.sylius.layout_resolver.target_type.single_taxon:
```

Comment:
Please keep these three definitions together, as is the case in the rest of the file.

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/17/commits/057d41669cd3128b87eaf8063b07be5151bc5382